pkg/virt-api/webhooks/validating-webhook/admitters/vmi-update-admitter.go

```diff
@@ -264,9 +263,8 @@ func admitVMILabelsUpdate(
 	oldVMI *v1.VirtualMachineInstance,
 	ar *v1beta1.AdmissionReview) *v1beta1.AdmissionResponse {
 
-	// Skip admission for internal components
-	allowed := webhooks.GetAllowedServiceAccounts()
-	if _, ok := allowed[ar.Request.UserInfo.Username]; ok {
+	if webhooks.IsKubeVirtServiceAccount(ar.Request.UserInfo.Username) {
+		// Skip admission for internal components
```

Comment:
I think we don't need this comment with the change of the function name.

Reply:
Right. Comments are evil.